extensions/mssql/src/objectExplorerNodeProvider/webhdfs.ts

```diff
@@ -0,0 +1,717 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
As discussed, please use the original license from WebHDFS and attribute to that repository. For example in the adapter.ts class in the prompts folder of this extension you'll see:
```
// This code is originally from https://github.com/DonJayamanne/bowerVSCode
// License: https://github.com/DonJayamanne/bowerVSCode/blob/master/LICENSE
`` 
You should point to the original github URL and use that license info.

Reply:
fixed